web/thesauruses/c/17/functions.json

```diff
@@ -0,0 +1,90 @@
+{
+    "meta": {
+        "language": "c",
+        "language_version": "17",
+        "language_name": "C",
+        "structure": "functions"
+    },
+    "concepts": {
+        "void_function_no_parameters": {
+            "code": "void func(){return;}",
+            "comment": "Return statement is optional.",
+            "name": "Function that does not return a value and takes no parameters"
+        },
+        "void_function_with_parameters": {
+            "code": "void foo(char c, int i, double d) { return; }",
+            "comment": "Return statement is optional.",
+            "name": "Function that does not return a value and that takes 1 or more defined parameters"
+        },
+        "void_function_with_keyword_parameters": {
+            "code": "void sum(int a=5,int b=7)",
+            "comment":"default paramaters",
```

Comment:
While I think adding in default parameters is probably a thing we should do (but it'd need to be on the meta language level so it works for all functions), adding it here is weird and doesn't fit with what the description actually says. Can you just make it `void sum(int a, int b)`?

Reply:
I think i misunderstood keyword parameters with default parameters .